UPDATING.md

```diff
@@ -24,6 +24,11 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Deprecation chain function after Airflow 1.10.2
```

Comment:
nit:  just called it `Deprecation chain function `

We don't know whether this change will be picked in the new release.

Reply:
WDYT

Reply:
Agree with that.